db/migrate/20160220123313_add_open_scap_result_entities.rb

```diff
@@ -0,0 +1,23 @@
+class AddOpenScapResultEntities < ActiveRecord::Migration
+  def up
+    create_table :openscap_results do |t|
+      t.belongs_to :container_image, :type => :bigint
```

Comment:
this also automatically creates an index if you use `t.belongs_to` or `t.references`